### DISCUSSION TITLE
Remove URL in Collection Header

### DISCUSSION THREAD

**Discussion by kaciccone** [knowledgefutures/pubpub#2178] [Support] [answered]
The URL is appearing in the Collection Header for this collection page: https://ncsu-wolfpack-solutions.pubpub.org/future-of-food. How can I remove it? When I click on "Edit Layout" for this collection I can't edit the Collection Header Block. This URL suddenly appeared on this page. However, it doesn't appear in the Collection Header for the other collection pages in this community (e.g., https://ncsu-wolfpack-solutions.pubpub.org/global-change).

**Comment by Dawit-Tegbaru** [comment, marked as answer]
Hello Karen! So happy to see you here, and great job on taking the Wolfpack Solutions community to the next level! 

Thanks for the heads up RE: the collection header block. We were able to remove the URL via the collection settings. However, I will file this issue as a bug for our engineers to look into further. I also could not replicate the case in other collections & communities. 

P.S. it doesn't look like past collections (Global Change and Pandemics) were deposited to Crossref. Nevertheless, please stay in touch if you intend to deposit the Future of Food so we can ensure the bug is fully resolved before proceeding.

**Comment by kaciccone** [comment]
Thanks for your help, Dawit. At this time we don't intend to deposit the collections to Crossref. I'll let you know if that changes.
